dev-db/mariadb-connector-odbc/mariadb-connector-odbc-3.1.6.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 2018-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils multilib-minimal
```

Comment:
Sorry for not noticing it earlier, but you should use `cmake` eclass instead. 

It just occurred to me there is `cmake-multilib` eclass also that could probably help you with the default phases below.

Reply:
Doesn't make a huge difference.  Looking at the eclass it basically just does what was done already, but updated since it's 7 fewer lines of code to maintain.